azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -184,8 +189,13 @@ public AzkabanWebServer(final Props props,
       DateTimeZone.setDefault(DateTimeZone.forID(timezone));
       logger.info("Setting timezone to " + timezone);
     }
-
     configureMBeanServer();
+
+    // Start Quartz Server
+    if(props.containsKey(ConfigurationKeys.ENABLE_QUARTZ) && props.getBoolean(ConfigurationKeys
+        .ENABLE_QUARTZ)) {
+      quartzScheduler.start();
```

Comment:
should we start the scheduler in constructor or prepareAndStartServer()?

Reply:
What's the benefits over that approach?

Reply:
to separate preparation work from constructor.

Reply:
Moving to prepareAndStartServer()